### ISSUE TITLE
✨  Add support for custom properties (homebrew)

### ISSUE THREAD

**Issue by RickCoxDev** [ebullient/ttrpg-convert-cli#349]
I have a homebrew that has a couple of custom properties for items. I would like to include these in the template.

```
"itemProperty": [
    {
		"name": "Herb",
		"source": "HHbH",
		"page": 5,
		"abbreviation": "Hrb",
		"template": "{@b Location:} {{item.customProperties.lcn}} {@b Requires Preparation:} {{item.customProperties.prep}}",
		"entries": []
	}
]
```
[Hamund's Herbalism Handbook](https://github.com/TheGiddyLimit/homebrew/blob/96bae769c6c7f6a19c16d16ed30c72b7526f408f/collection/Jasmine%20Yang%3B%20Hamund's%20Herbalism%20Handbook.json#L6396)

Here is an example of an item with these properties:
```
{
	"name": "Sentry Bush",
	"source": "HHbH",
	"page": 30,
	"type": "Herb",
	"rarity": "common",
	"value": 100,
	"property": [
		"Hrb"
	],
	"entries": [
		{
			"type": "entries",
			"name": "Description:",
			"entries": [
				"A scraggly brush found only on high altitudes, its bitter twigs produce an incredibly potent stimulant that when brewed into a tea, can completely replace one's need to sleep. Due to its inaccessible location, but widely sought-after use, mountain clans are often found carrying them in their caravan to trade to people that need to stay awake for long periods of time."
			]
		},
		{
			"type": "entries",
			"name": "Effect:",
			"entries": [
				"After preparing this herb, you gain all the benefits of 8 hours of sleep (or the equivalent form of dormancy for your race). You still need to rest for at least 8 hours in order to benefit from a long rest."
			]
		}
	],
	"customProperties": {
		"lcn": "Mountain",
		"prep": "Yes"
	},
	"foundryType": "consumable"
}
```
[Hamund's Herbalism Handbook](https://github.com/TheGiddyLimit/homebrew/blob/96bae769c6c7f6a19c16d16ed30c72b7526f408f/collection/Jasmine%20Yang%3B%20Hamund's%20Herbalism%20Handbook.json#L4927)

**Comment by ebullient** [comment]
Item properties are absolutely already taken into account provided they are in the right format in _meta

**Comment by RickCoxDev** [comment]
@ebullient the properties for "Location" and "Requires Preparation".
![image](https://github.com/ebullient/ttrpg-convert-cli/assets/6799456/42eba0c2-18c7-4ea0-ad4e-9b5bc3bd0e73)
```
"customProperties": {
	"lcn": "Mountain",
	"prep": "Yes"
}
```

The `lcn` and `prep` keys.

**Comment by ebullient** [comment]
ok. A different kind of custom properties. ;) 

Herb is, in and of itself, a custom Item property.

**Comment by revilowaldow** [comment]
> Always item template usage?

Not necessarily, if you wanted a stress test try https://github.com/TheGiddyLimit/homebrew/blob/master/magicvariant/aeyana%3B%20Gemstones.json
Which includes `customProperties` mixed with `itemEntry` and `variant` generation.
Clock the `hasRefs` flag.

![image](https://github.com/ebullient/ttrpg-convert-cli/assets/52298102/667eb2c1-8998-4232-9f2b-02e0fdd1e745)

![image](https://github.com/ebullient/ttrpg-convert-cli/assets/52298102/cd1c89a2-5d8d-4500-adbd-d8e92414af71)

Basically this one '''item''' explodes to 46 items on the site

![image](https://github.com/ebullient/ttrpg-convert-cli/assets/52298102/3c2e6432-b234-476b-ae66-3c793eeb7458)
See video:
https://github.com/ebullient/ttrpg-convert-cli/assets/52298102/9a0a3e25-8329-413e-aeee-f88c3a14693b
